src/components/Card/Card.stories.tsx

```diff
@@ -59,3 +74,181 @@ export const Dragging: StoryObj<Args> = {
     elevation: 'dragging',
   },
 };
+
+export const ButtonActionCalloutCard: StoryObj<Args> = {
+  args: {
+    elevation: 'raised',
+  },
+  parameters: {
+    badges: ['1.0', 'implementationExample'],
+  },
+  render: (args) => (
+    <Card
+      className={buttonActionCalloutStyles['button-action-callout-card']}
+      {...args}
+    >
+      <Card.Body>
+        <Heading as="h3" size="h5">
+          Do This Checkpoint
+        </Heading>
+        <div
+          className={
+            buttonActionCalloutStyles['button-action-callout-card__body-inner']
+          }
+        >
+          Develop the text of your Body Book, crafting evidence-supported
+          explanations on how the body is organized and its functions.
+          <div>
+            <Button variant="primary">Preview</Button>
+          </div>
+        </div>
+      </Card.Body>
+    </Card>
+  ),
+};
+
+export const CardWithNotification: StoryObj<Args> = {
```

Comment:
fyi some files in the "Recipes" section still show up. not sure if you meant to delete them all here

<img width="221" alt="Screenshot 2023-08-02 at 2 33 58 PM" src="https://github.com/chanzuckerberg/edu-design-system/assets/15840841/905a7aa3-fe66-47a9-a122-d1c634544599">